helm/values-dev.yaml

```
@@ -172,6 +172,7 @@ microservice-chart:
     NODE_FORWARDER_CONNECTION_TIMEOUT: "10000"
     REDIRECT_PAYMENT_TYPE_CODES: "BPPIITRRXXX-RBPR,BPPIITRRXXX-RBPP,BPPIITRRXXX-RBPB,PPAYITR1XXX-RBPB,CHECKOUT-PPAYITR1XXX-RBPP,IO-PPAYITR1XXX-RBPP,PPAYITR1XXX-RBPR,RBPS,RPIC,RICO"
     NPG_REFUND_DELAY_FROM_AUTH_REQUEST_MINUTES: "10"
+    NPG_REFUND_DELAY_FOR_REFUND_OPERATIONS_SECONDS: "10"
```

```suggestion
    NPG_REFUND_EVENT_PROCESSING_DELAY_SECONDS: "10"
```

what about this name?